ledger/participant-state/kvutils/tools/BUILD.bazel

```
@@ -47,11 +47,18 @@ da_scala_binary(
 da_scala_test(
     name = "integrity-check-tests",
     size = "small",
-    srcs = glob(["integrity-check/src/test/scala/ledger/participant/state/kvutils/tools/integritycheck/IntegrityCheckerSpec.scala"]),
+    srcs = glob(["integrity-check/src/test/scala/ledger/participant/state/kvutils/tools/integritycheck/**/*.scala"]),
```

You can simplify this to:
```suggestion
    srcs = glob(["integrity-check/src/test/scala/**/*.scala"]),
```

---

Yes, I will do so in the next PR of the chain.